pkg/api/norman/customization/aks/handler.go

```diff
@@ -100,6 +101,13 @@ func (h *handler) ServeHTTP(writer http.ResponseWriter, req *http.Request) {
 	var err error
 
 	switch resourceType {
+	case "aksUpgrades":
+		if serialized, errCode, err = listKubernetesUpgradeVersions(req.Context(), h.clusterLister, capa); err != nil {
+			logrus.Errorf("[aks-handler] error getting kubernetes upgrade versions: %v", err)
```

Comment:
There are a couple things that might be referred to as the aks-handler, could call this aksUpgrades handler or something.